tests/integration/Storage/Serializer/Msgpack/SerializeCest.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+/**
+ * This file is part of the Phalcon Framework.
+ *
+ * (c) Phalcon Team <team@phalcon.io>
+ *
+ * For the full copyright and license information, please view the LICENSE.txt
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Phalcon\Tests\Integration\Storage\Serializer\Msgpack;
+
+use Codeception\Example;
+use Phalcon\Storage\Serializer\Msgpack;
+use stdClass;
+use UnitTester;
+
+use function msgpack_pack;
+
+class SerializeCest
+{
+    /**
+     * Tests Phalcon\Storage\Serializer\Msgpack :: serialize()
+     *
+     * @dataProvider getExamples
+     *
+     * @param UnitTester $I
+     * @param Example    $example
+     *
+     * @author Phalcon Team <team@phalcon.io>
+     * @since  2020-09-09
+     */
+    public function storageSerializerMsgpackSerialize(UnitTester $I, Example $example)
```

Comment:
https://github.com/phalcon/cphalcon/pull/15155

Running msgpack integration tests (for example `php vendor/bin/codecept run tests/integration/Storage/Serializer/Msgpack/SerializeCest.php`) were outputting special characters to the console and froze it inside nanobox, so I had to remove the `@dataProvider` to fix this.